hoot-core/src/main/cpp/hoot/core/elements/Node.h

```diff
@@ -118,7 +118,7 @@ class Node : public Element
 
   void setY(double x);
 
-  virtual ElementType getElementType() const { return ElementType::Node; }
+  virtual ElementType getElementType() const { return ElementType(ElementType::Node); }
```

Comment:
@mschicker What is going on here?  Isn't `ElementType::Node` an `ElementType`?  Why the change?

Reply:
Funny, huh?
`ElementyType::Node` is an enumeration value. 
But `ElementType` is actually a class. I think some sort of default constructing was going on, but it was causing me heartburn.

Reply:
Ok, no worries.  Sometimes things get changed ever so slightly and the weirdest things have to be changed to keep some code working.  I'm good with the change.